pyproject.toml

```diff
@@ -7,7 +7,6 @@ name = "pygmt"
 description = "A Python interface for the Generic Mapping Tools"
 readme = "README.rst"
 requires-python = ">=3.9"
-license = {text = "BSD License"}
```

Comment:
See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license

> If you are using a standard, well-known license, it is not necessary to use this field. Instead, you should one of the [classifiers](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#classifiers) starting with License ::. (As a general rule, it is a good idea to use a standard, well-known license, both to avoid confusion and because some organizations avoid software whose license is unapproved.)